volontulo/templates/common/footer.html

```diff
@@ -26,6 +26,7 @@
             <h3>Oferty</h3>
             <ul>
                 <li><a href="{% url 'offer_apply' 1 %}">Zgłoś chęć współpracy przy ofercie</a></li>
+                <li><a href="{% url 'show_offer' 1 %}">Podgląd oferty</a></li>
```

Comment:
this should be most likely the first item coz to apply for offer you rsther see it first